apps/minifront/src/state/dutch-auction/constants.ts

```diff
@@ -0,0 +1,26 @@
+export const DURATION_OPTIONS = ['10min', '30min', '1h', '2h', '6h', '12h', '24h', '48h'] as const;
+export type DurationOption = (typeof DURATION_OPTIONS)[number];
+
+/**
+ * Blocks are ~5 seconds long (so, 12 blocks/minute), and the minimum duration
+ * for an auction is 10 minutes (so, 120 blocks). All other auction durations
+ * are even multiples of 10 minutes. So we'll set the step count to 120, so that
+ * the sub-auctions can divide evenly into the number of intervening blocks.
+ */
+export const STEP_COUNT = 120n;
```

Comment:
Reviewers, let me know what you think of this logic.

Per @hdevalence 's comment [here](https://hackmd.io/jncILV-KS3Cqtawco71dhA?comment=14e8b2a5-db62-4f17-b9ef-784b3561f5ef&utm_source=comment-card&utm_medium=icon), a step count of ~100 is a good target. Since we're starting at a duration of 10 minutes, which doesn't divide evenly into 100 blocks (since blocks are added roughly every 5 seconds), I hard-coded the step count at 120.

Henry also wrote:
> Longer time should mean more sub-auctions and slower price declines.

Not sure I understand this — does the number of steps influence how the price declines? And if a longer time = more sub-auctions, what should our algorithm be for increasing step count as duration increases?

(In the meantime, though, I figured this is a mergeable first effort. Let me know if you think otherwise.)